java/client/src/main/java/glide/api/models/configuration/Route.java

```diff
@@ -0,0 +1,48 @@
+package glide.api.models.configuration;
+
+import lombok.Builder;
+import lombok.Getter;
+
+/** Request routing configuration. */
+@Builder
+@Getter
+public class Route {
+
+    public enum RouteType {
+        /** Route request to all nodes. */
+        AllNodes,
```

Comment:
Feels like we should follow the convention to capitalize all enums.  No?

Reply:
Fixed in 6cbc27f5